by-dataframe/polars/write_sqlalchemy.py

```diff
@@ -1,5 +1,17 @@
 """
-Import a Parquet file into CrateDB using Polars and SQLAlchemy.
+Efficient data imports using CrateDB, Polars, and SQLAlchemy.
+
+The `insert_bulk` utility provides efficient bulk data transfers
+when using dataframe libraries like pandas, Dask, and Polars.
+
+You will observe that the optimal chunk size highly depends on the
+shape of your data, specifically the width of each record, i.e. the
+number of columns and their individual sizes, which will in the end
+determine the total size of each batch/chunk.
+
+https://cratedb.com/docs/sqlalchemy-cratedb/support.html#bulk-support-for-pandas-and-dask
+https://cratedb.com/docs/sqlalchemy-cratedb/dataframe.html#efficient-insert-operations-with-pandas
+https://cratedb.com/docs/crate/reference/en/latest/interfaces/http.html#bulk-operations
```

Comment:
Maybe use a subsequent opportunity to reorganize/bundle those resources, which share the same ingredients between pandas/Dask/Polars, specifically when using SQLAlchemy. It makes sense to decrease redundancy, and generalize across all of the items of the same category.